src/tests/functional/fixtures/112_hashes.csv

```diff
@@ -0,0 +1,112 @@
+00a16c806ff694b64e566886bba5122655eff89b45226cddc8651df7860e4524
```

Comment:
What's the difference between 112_hashes.csv and hashes.csv? Do we need both files?

Reply:
Nope, hashes.csv is going to be deleted